src/main/java/org/opensearch/flowframework/util/ParseUtils.java

```diff
@@ -356,4 +372,17 @@ private static Object conditionallySubstitute(Object value, Map<String, Workflow
         }
         return value;
     }
+
+    /**
+     * Generates a string based on an arbitrary String to object map sing Jackson
+     * @param map content map
+     * @return instance of the string
+     * @throws JsonProcessingException JsonProcessingException from Jackson for issues processing map
+     */
+    public static String parseArbitraryStringToObjectMapToString(Map<String, Object> map) throws JsonProcessingException {
```

Comment:
This functionality looks like it should probably exist in core, such as in `o.o.core.common.Strings` util class, or in OpenSearch's `o.o.index.mapper.ObjectMapper` class.   Jackson is a regular source of CVEs and we've made efforts to remove parsing like this from core for good reasons.

I'm OK with merging this now but can we open an issue to investigate other alternatives?

Reply:
I see Jackson is used in core in lots of other plugins so I thought this would be fine, but I can open an issue to investigate